### PR TITLE
Add log notice about loaded lua scripts

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: common/redisapi.h [hunk 1]
```diff
@@ -12,7 +12,13 @@ static inline std::string loadRedisScript(DBConnector* db, const std::string& sc
     RedisCommand loadcmd;
     loadcmd.format("SCRIPT LOAD %s", script.c_str());
     RedisReply r(db, loadcmd, REDIS_REPLY_STRING);
-    return r.getReply<std::string>();
+
+    std::string sha = r.getReply<std::string>();
+
+    SWSS_LOG_NOTICE("lua script loaded, sha: %s", sha.c_str(), script.c_str());
+    SWSS_LOG_INFO("lua script content, sha: %s", script.c_str());
```
Comment (qiluo-msft) [inline common/redisapi.h]: SWSS_LOG_INFO("lua script: %s", script.c_str());

**Comment by lguohan** [comment]
test

**Comment by lguohan** [comment]
test

**Comment by lguohan** [comment]
test again

**Comment by kcudnik** [comment]
addressed comments

**Comment by kcudnik** [comment]
test

**Pull request merged**